src/shared/logger/index.ts

```diff
@@ -11,6 +11,7 @@ export { showLogOutputChannel } from './outputChannel'
 
 export type Loggable = loggableType.Loggable
 
+//TODO: add a logOnce() to the Logger interface since this is a common pattern
```

Comment:
good thought, but not the right place for this comment and will get lost/forgotten. We can track it as an issue instead